WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -1157,18 +1157,4 @@ private static void updateIsBookmarkedField(final ReaderPostList posts) {
     private static ReaderBlogIdPostIdList getBookmarkedPostIds() {
         return getBlogIdPostIdsWithTagType(ReaderTagType.BOOKMARKED, 99999);
     }
-
-    /**
-     * getBlogIdForFeed is a helper method that will query the post table for a post with the requested
-     * feedId. It will return a limit of 1
-     * @param feedId
-     * @return The associated blogId
-     */
-    public static Long getBlogIdForFeed(long feedId) {
```

Comment:
This was code was added in the first PR and can safely be removed because it is not used.